sweep/sweeper.go

```diff
@@ -237,12 +237,21 @@ type UtxoSweeper struct {
 	wg   sync.WaitGroup
 }
 
+// feeDeterminor defines an alias to the function signature of
```

Comment:
Should be `Determiner` to be "dictionary correct".

Reply:
fixed!